upstream-community-operators/infinispan/infinispan-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,181 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: infinispan-operator.v0.1.0
+  namespace: placeholder
+  annotations:
+    tectonic-visibility: ocs
+    capabilities: Basic Install
+    categories: "Database"
+    alm-examples: |
+      [
+        {
+          "apiVersion": "infinispan.org/v1",
+          "kind": "Infinispan",
+          "metadata": {
+            "name": "example-infinispan"
+          },
+          "spec": {
+            "size": 3
+          }
+        },
+        {
```

Comment:
Not sure how OLM handles multiple examples for the same CR. @alecmerdler do you have some insight into this?

Reply:
Console will just grab the first one for now.

Reply:
@SamiSousa etcd 0.9.2 operator shows multiple examples, I got the idea from there.